docs/jwst/firstframe/arguments.rst

```diff
@@ -1,4 +1,9 @@
 Step Arguments
 ==============
 
-The first frame correction has no step-specific arguments.
+The ``firstframe`` step has the following step-specific arguments.
+
+``--bright_use_group1`` (boolean, default=False)
+    If True, setting the group 1 groupdq to DO_NOT_USE will not be done 
+    for pixels that have the saturation flag set for group 3, but not group 2.
+    This will allow a slope to be determined for this pixel.
```

Comment:
It might be helpful to add a note here about when this is appropriate (i.e. for very bright targets).

Reply:
Will do.  It could be used for *all* data, just will only have an impact for those pixels with really bright sources.  For all other pixels, the firstframe flagging will happen just as it current does.  Hopefully this is clear, but maybe I should say this explicitly?  Please advise.

Reply:
Yes, I think it would help to be explicit here - it was a little surprising to me at first that turning it on made no difference when I tested it on data with no very bright sources.